src/components/timeline/TrackScreenshots.js

```diff
@@ -263,9 +263,10 @@ class ScreenshotStrip extends PureComponent<ScreenshotStripProps> {
     const timeToPixel = time =>
       outerContainerWidth * (time - rangeStart) / rangeLength;
 
+    const leftmostPixel = Math.max(timeToPixel(screenshots[0].start), 0);
     let screenshotIndex = 0;
     for (
-      let left = timeToPixel(screenshots[0].start);
+      let left = leftmostPixel;
```

Comment:
I think this worked properly when you first implemented it, and this broke when we added the screenshot "right before the start of the range".

Reply:
Ah, makes sense.